packages/matter-node.js/README_EXAMPLES.md

```diff
@@ -61,12 +61,13 @@ The following parameters are available:
 * -announceinterface: limit mdns announcements to the provided network interface, e.g. "en0" (default: all interfaces available)
 * -port: the port to listen on for the device (default: 5540)
 * -store: the storage location (directory) to use for storing the pairing information (default: device-node). Delete the directory or provide an alternative name to reset the device
+* -clearstorage: the storage location will be reset on start of the process
```

Comment:
In general, common norms have double-dash full-word parameters and synonymous single-dash letter parameters:
`--port` or `-p`.

Not blocking this PR on this, but wanted to mention this feature request will reviewing.

Reply:
You are right, but the here used minimalistic self-brews commandline parameter handling do not support this.